include/tvm/data_layout.h

```diff
@@ -210,6 +210,20 @@ class Layout : public NodeRef {
     return ct;
   }
 
+  /*! \return Concatenation of all primal axes */
+  inline std::string get_primal_axes() const {
```

Comment:
suggest to wrap it as `Layout` so later we can deal with Layout instead of std::string